TriangleApp/app/src/main/res/xml/preference.xml

```diff
@@ -2,122 +2,102 @@
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
 
     <PreferenceCategory
-        android:title="Identity"
+        android:title="@string/category_id"
         android:key="category_identity">
 
-        <!--<CheckBoxPreference-->
-            <!--android:key="perform_updates"-->
-            <!--android:summary="Enable or disable data updates"-->
-            <!--android:title="Enable updates"-->
-            <!--android:defaultValue="true"-->
-
-            <!--/>-->
-
         <EditTextPreference
             android:key="username"
-            android:title="Display name"
-            android:summary="This will be shown to other users"
-            android:dialogTitle="Display name"
-            android:dialogMessage="Fill in your name that will be shown to viewers."
+            android:title="@string/display_name"
+            android:summary="@string/username_summary"
+            android:dialogTitle="@string/display_name"
+            android:dialogMessage="@string/username_dialog_message"
             android:defaultValue="Anoniem" />
```

Comment:
defaultvalues zijn aanpasbare values, die kunnen worden opgeslagen in de preferences, net als keys, kunnen deze niet in de strings.xml